File: AnDsSistema/2023 Semestre 1/3a-Lab/Aula12/aula12Veiculo/src/Veiculo.java
public abstract class Veiculo{

    String marca;
    String modelo;
    int ano;

    public abstract void ligarMotor();
    public abstract void desligarMotor();

    Veiculo(String _marca, String _modelo, int _ano){

        this.marca = _marca;
        this.modelo = _modelo;
        this.ano = _ano;
    }
}

File: AnDsSistema/2023 Semestre 1/4a-OOP/Aula16+Refator/AP2_Refatorar_Estacionamento/Domain/Entities/Veiculo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AP2_Refatorar_Estacionamento
{
    public abstract class Veiculo
    {

        public int Id {get;set;}
        public string Placa {get;set;}   
        public string Modelo {get;set;}
        public double Combustivel {get;set;}
    }
}